setup.py

```diff
@@ -11,23 +11,24 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""Setup
-"""
+version = '1.6.0'
```

Comment:
Yes.  

@jimfulton looks like you didn't push the commits / tag for the 2016-05-21 1.6.0 release.  Maybe need to do that, then rebase this on top?